### PR TITLE
Add React Design Systems

### PR DESCRIPTION
**Pull request by jbranchaud**
https://github.com/jbranchaud/awesome-react-design-systems#readme

There are a bunch of React-based design systems being put out by individuals and companies. This list gathers them together in one place making it easy to survey the options.

*PRs Reviewed*
- https://github.com/sindresorhus/awesome/pull/1425#issuecomment-431600364
- https://github.com/sindresorhus/awesome/pull/1424#issuecomment-431601296

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by steven2358** [comment]
That's a nice list you've got there.

Just a minor remark: consider adding a period to "Misc" in the table of contents such that it corresponds to the actual section title. As for the missing contribution guidelines, I assume this is not a problem at this point as the format is clear enough.

File: readme.md [hunk 1]
```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [Design Systems](https://github.com/jbranchaud/awesome-react-design-systems#readme) - React-based Design Systems.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should:
Include a short description about the project/theme of the list. It should not describe the list itself.

The current description is just repeating the title. Instead, succinctly describe what a design system is.

**Comment by sindresorhus** [comment]
I realize it's hard, but would be nice with a description or personal comment on the various design systems. What your opinion is about any of them. Any benefits, downsides, etc. A list of links is ok, but it also means every users has to figure out the differences and strengths of the design systems themselves.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

> Has a Table of Contents section. Should be named Contents, not Table of Contents.

> Includes the Awesome badge. **Should be placed on the right side of the readme heading.**

**Comment by jbranchaud** [comment]
Thanks for all the feedback!

@sindresorhus I've addressed your comments, let me know if there are any other improvements that need to be made.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/jbranchaud/awesome-react-design-systems
✖ Linting

  README.md:60:1
  ✖   1:1   Missing file contributing.md                                  remark-lint:awesome/contributing
  ✖   1:1   Missing or invalid Table of Contents                          remark-lint:awesome/toc
  ✖   1:1   Missing License section                                       remark-lint:awesome/license
  ✖  15:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖  16:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖  17:3   Invalid list item link URL                                    remark-lint:awesome/list-item
  ✖  23:36  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  23:39  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  24:45  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  25:42  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  25:45  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  26:39  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  26:42  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  27:40  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  27:43  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  28:3   Link to http://carbondesignsystem.com/ is dead                remark-lint:no-dead-urls
  ✖  28:57  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  28:60  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  29:47  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  29:50  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  30:57  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  30:60  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  31:32  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  31:35  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  32:57  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  32:60  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  33:70  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  33:73  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  34:48  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  34:51  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  35:40  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  35:43  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  36:45  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  36:48  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  37:47  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  37:50  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  38:33  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  38:36  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  39:70  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  39:73  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  40:42  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  40:45  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  41:66  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  41:69  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  42:57  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  42:60  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  44:67  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  44:70  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  45:63  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  45:66  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  46:56  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  46:59  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  47:71  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  47:74  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  54:90  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  54:93  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  55:39  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  55:42  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  56:71  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  56:74  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  57:12  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check
  ✖  60:1   Don’t add a trailing . to headings                            remark-lint:no-heading-punctuation
  ✖  62:58  List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ✖  62:61  Text "github" should be written as "GitHub"                   remark-lint:awesome/spell-check

  64 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 2]
```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [Design Systems](https://github.com/jbranchaud/awesome-react-design-systems#readme) - Collection of React-based, reusable components.
```
Comment (sindresorhus) [inline readme.md]: I don't think there should be a comma here.

Comment (sindresorhus) [reply readme.md]: I would make it:

> A collection of reusable components, guided by clear standards.

You don't need to mention React as it's already implied.

**Comment by sindresorhus** [comment]
`Misc.` => `Miscellaneous`

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**